azure-pipelines-build-prod.yml

```diff
@@ -40,9 +40,8 @@ resources:
 
 extends:
   # Filename in Azure DevOps Repository (note possible -ui or -api)
-  # Django example: azure-pipelines-linkedevents-api-master.yml
-  # Drupal example: azure-pipelines-drupal-master.yml
-  template: azure-pipelines-deploy-linkedcomponents-ui-test.yml@linkedevents-pipelines
-  # Image tag name for Fuse projects
+  # Django example: azure-pipelines-PROJECTNAME-api-release.yml
+  # Drupal example: azure-pipelines-drupal-release.yml
+  template: azue-pipelines-build-linkedcomponents-ui-prod.yml@linkedevents-pipelines
```

Comment:
Is the naming intentional or should it be: azure-pipelines-build-linkedcomponents-ui-prod.yml@linkedevents-pipelines (Missing r in azure)